test/e2e/networks/simple.toml

```diff
@@ -1,5 +1,16 @@
+# ipv6 = false
+# initial_height = 0
+# key_type = "ed25519"
+# evidence = 10
+# log_level = ""
+# queue_type = "priority"
+disable_legacy_p2p = true
+
 [node.validator01]
+  disable_legacy_p2p = true
 [node.validator02]
+  disable_legacy_p2p = true
 [node.validator03]
+  disable_legacy_p2p = true
 [node.validator04]
-
+  disable_legacy_p2p = true
```

Comment:
Maybe revert this back to what it was. I guess at some point we will make the transition to defaulting with the new p2p stack